packages/react/src/components/ComboBox/ComboBox-test.js

```diff
@@ -234,5 +236,35 @@ describe('ComboBox', () => {
       expect(firstListBox()).toBeEmptyDOMElement();
       expect(secondListBox()).toBeEmptyDOMElement();
     });
+    it('should open menu without moving focus on pressing Alt+ DownArrow', async () => {
```

Comment:
@tay1orjones I have added some test cases but they are not working correctly, glad if you could help

Reply:
@tay1orjones ping !